pkg/virt-handler/device-manager/generic_device.go

```diff
@@ -176,6 +183,7 @@ func (dpi *GenericDevicePlugin) Start(stop chan struct{}) (err error) {
 func (dpi *GenericDevicePlugin) Stop() error {
 	defer close(dpi.done)
 	dpi.server.Stop()
+	dpi.initialized = false
```

Comment:
This would also have to be written to by the lock. Maybe create helpers `setInitialized` and `getInitialized` which use a lock, just for clarity, always when you change or read it (even on Start).